projects/kit/components/multi-select/native-multi-select/native-select.template.html

```diff
@@ -0,0 +1,13 @@
+<ng-container *ngIf="control.control?.valueChanges | async"></ng-container>
+
+<tui-data-list-wrapper
+    *tuiDataList
+    [items]="items"
+></tui-data-list-wrapper>
+<option
+    *ngFor="let option of items"
+    [attr.selected]="option | tuiMapper : selectedMapper : host.value"
+    [value]="option"
```

Comment:
We should also allow items to be disabled. And add it to the demo stand please. This applies to plain Select too.

Reply:
I'll fix plain select in a separate PR https://github.com/Tinkoff/taiga-ui/issues/4005